src/shutdown/umount.c

```diff
@@ -522,7 +522,7 @@ static int remount_with_timeout(MountPoint *m, int umount_log_level) {
         if (r < 0)
                 return r;
         if (r == 0) {
-                log_info("Remounting '%s' read-only in with options '%s'.", m->path, m->remount_options);
+                log_info("Remounting '%s' read-only with options '%s'.", m->path, strempty(m->remount_options));
```

Comment:
Wouldn't strna be better? Empty strings in log lines are often confusing, as you need to know there's supposed to be something there